pkg/rest/core/fake/rest_client.go

```diff
@@ -188,13 +194,110 @@ func getRouter(storage NamespacedStorage) http.Handler {
 		deleteItem(storage),
 	).Methods("DELETE")
 	r.HandleFunc(
+		"/apis/servicecatalog.k8s.io/v1alpha1/watch/namespaces/{namespace}/{type}/{name}",
+		watchItem(watcher),
+	)
```

Comment:
Should we constrain this to the HTTP `GET` method. All the other routes above are constrained by method. Same question applies for the route below this one.